server/controllers/projectController.js

```diff
@@ -0,0 +1,59 @@
+const Project = require("../models/projects");
+const User = require("../models/User");
+
+async function getAllProjects(req, res) {
+  const projects = await Project.find().sort({ name: 1 });
+  if (!projects) {
+    return res.status(404);
+  }
+  res.send(projects);
+}
+
+async function getProjectsForUser(req, res) {
+  const projects = await User.find({ _id: req.params.userId })
+    .select({ username: 1 })
+    .populate("projects", "title");
```

Comment:
This might be thinking too much in the future, but if this is a public route, then maybe we should only return live projects - I think it's okay not to handle this, but we should consider for the future

Reply:
Sure, will leave this feature for later changes.